pyuvdata/uvdata/uvdata.py

```diff
@@ -3715,9 +3777,9 @@ def _key2inds(self, key):
 
         Returns
         -------
-        blt_ind1 : ndarray of int
-            blt indices for antenna pair.
-        blt_ind2 : ndarray of int
+        blt_ind1 : ndarray of int or slice or None
+            blt indices for antenna pair. None if antpair is not in data.
```

Comment:
It looks like it raises an error if the antpair isn't in the data? Can you add a `Raises` section to this docstring?

Reply:
Ah yeah this was a bit confusing. Behaviour is that if the baseline in a particular direction (e.g. (1,0,'xx')) does not exist, it returns None for that direction, but if they BOTH don't exist (so also (0, 1, 'xx') doesn't exist) then an error is raised. I added a Raises section as you suggest.